landing-page/content/common/vendors.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Vendors"
+url: vendors
+disableSidebar: true
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+## Vendors Supporting Iceberg Tables
+
+This page contains some of the vendors who are shipping and supporting Apache Iceberg in their products
+
+
+### [Cloudera](http://clouera.com)
+
+Cloudera Data Platform integrates Apache Iceberg to the following components:
+* Apache Hive, Apache Impala, and Apache Spark to query Apache Iceberg tables
+* Cloudera Data Warehouse service providing access to Apache Iceberg tables through Apache Hive and Apache Impala
+* Cloudera Data Engineering service providing access to Apache Iceberg tables through Apache Spark
+* The CDP Shared Data Experience (SDX) provides compliance and self-service data access for Apache Iceberg tables
+* Hive metastore, which plays a lightweight role in providing the Iceberg Catalog
+* Data Visualization to visualize data stored in Apache Iceberg
+
+https://docs.cloudera.com/cdp/latest/cdp-iceberg/topics/iceberg-in-cdp.html
+
+### [Starburst](http://starburst.io)
+
+Starburst is a commercial offering for the [Trino query engine](https://trino.io). Trino is a distributed MPP SQL query engine that can query data in Iceberg at interactive speeds. Trino also enables you to join Iceberg tables with an [array of other systems](https://trino.io/docs/current/connector.html). With Starburst, we offer both an [enterprise deployment](https://www.starburst.io/platform/starburst-enterprise/) and a [fully managed service](https://www.starburst.io/platform/starburst-galaxy/) to make managing and scaling Trino a flawless experience. We also provide customer support and house many of the original contributors to the open-source project that know Trino best. Learn more about [our Iceberg connecter](https://docs.starburst.io/latest/connector/iceberg.html).
```

Comment:
@bitsondatadev, could you remove personal pronouns from this? I think "Starburst offers ..." is more clear that "With Starburst, we offer ..." because it isn't clear who "we" is, and I don't want there to be confusion that the Iceberg community endorses a particular vendor.